service/msteams/ms_teams.go

```diff
@@ -0,0 +1,59 @@
+package msteams
+
+import (
+	goteamsnotify "github.com/atc0005/go-teams-notify/v2"
+	"github.com/pkg/errors"
+)
+
+// MSTeams struct holds necessary data to communicate with the MSTeams API.
+type MSTeams struct {
+	client   goteamsnotify.API
+	webHooks []string
+}
+
+// New returns a new instance of a MSTeams notification service.
+// For more information about telegram api token:
+//    -> https://github.com/atc0005/go-teams-notify#example-basic
+func New(apiToken string) *MSTeams {
```

Comment:
@nikoksr `apiToken` is not required here all authentication for this service is done using the webhook urls which gets injected using the `AddReceivers` function.

Reply:
Good catch!